packages/headless/src/helper/helper.module.ts

```diff
@@ -1,22 +1,21 @@
-import { Global, Module } from "@nestjs/common";
-import { Helper } from "./helper.interface";
-import { HelperService } from "./helper.service";
-import { ServiceResponse } from "./serviceResponse";
-import { IServiceResponse } from "./serviceResponse/service.response.interface";
+import { Global, Module } from '@nestjs/common';
+import { Helper } from './helper.interface';
+import { HelperService } from './helper.service';
+import { ServiceResponse } from './serviceResponse';
+import { IServiceResponse } from './serviceResponse/service.response.interface';
 
 @Global()
 @Module({
-    providers: [
-        {
-            provide: IServiceResponse,
-            useClass: ServiceResponse
-        },
-        {
-            provide: Helper,
-            useClass: HelperService
-        }
-    ],
-    exports: [Helper]
+  providers: [
+    {
+      provide: IServiceResponse,
+      useClass: ServiceResponse,
+    },
+    {
+      provide: Helper,
+      useClass: HelperService,
+    },
+  ],
+  exports: [Helper, IServiceResponse],
```

Comment:
I dont think any change should be done here. If you check the base branch, everything should be okay.